src/server.c

```diff
@@ -1335,18 +1379,24 @@ int serverCron(struct aeEventLoop *eventLoop, long long id, void *clientData) {
                 used = kvstoreSize(server.db[j].keys);
                 vkeys = kvstoreSize(server.db[j].expires);
                 if (used || vkeys) {
-                    serverLog(LL_VERBOSE, "DB %d: %lld keys (%lld volatile) in %lld slots HT.", j, used, vkeys, size);
+                    serverLog(LL_VERBOSE, "DB %d: Total %lld keys (%lld/%lld volatile keys) in %lld slots HT.", j, used,
+                              vkeys, used, size);
```

Comment:
```suggestion
                    serverLog(LL_VERBOSE, "DB %d: %lld keys (%lld volatile) in %lld slots HT.", j, used, vkeys, size);
```
I'm not really convinced this adds value, you're just basically printing the used value twice.

Reply:
Thank you for your review.